LINK/bin/miqnet.sh

```diff
@@ -279,13 +281,12 @@ GATEWAY=${3}
 NM_CONTROLLED=no
 DNS1=${4}" >> ${cfg}.tmp
   mv ${cfg}.tmp $cfg
+  restorecon $cfg
   log "set_redhat_static: cfg new: $(cat ${cfg})"
 
   # Truncate the temporary io file
   :> $TMP_IO
 
-  /sbin/restorecon -R /etc/sysconfig
```

Comment:
There is nothing else in sysconfig that we are editing from miqnet.sh that needs to be changed here.

If there is some other file that we are touching somewhere else, we really shouldn't be relying on miqnet to change the permissions after the fact.